### PR TITLE
Basic example for MarkdownDB tutorial

### PR DESCRIPTION
**Pull request by olayway**
Related to: #15 

## Changes
- example node app using MarkdownDB created for the purpose of the tutorial: https://hackmd.io/FLH_XNjnRvqRBtN6k7KibQ?edit

### CODE REVIEW

**Review by luccasmmg — COMMENTED** [review]
I've added a few comments to the hackmd, as it was easier to read there

**Comment by rufuspollock** [comment]
@olayway why not include the tutorial itself as the README.md in the example folder or similar - i think much better to have the tutorial in PR unless special reason not to ... (in general we don't review hackmd's as discussed b/c better to use GH PR review tooling etc)

**Comment by olayway** [comment]
@rufuspollock I've copied over the tutorial to this example's README

@luccasmmg thank you for the review, I've applied all your suggestions. Can I merge now ? :)

File: examples/basic-example/README.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+```bash
+mkdir projects
+cd projects
+touch portaljs.md flowershow.md datapipes.md giftless.md data-cli.md aircan.md
+```
+
+In each file we'll write some short info about a given project, like so:
+
+```md
+# What is Portal.JS
```
Comment (rufuspollock) [inline examples/basic-example/README.md:18]: Very clever 👍 

BTW you could also have the default project be markdowndb (or one of the other ones). That way people learn about markdowndb as part of the tutorial

**Pull request merged**